proxies/sandbox/apiproxy/policies/JavaScript.MessageBatches.Create.MapMessages.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Javascript async="false" continueOnError="false" enabled="true" timeLimit="200" name="JavaScript.MessageBatches.Create.MapMessages">
+    <DisplayName>JavaScript.MessageBatches.Create.MapMessages</DisplayName>
+    <Properties/>
+    <Source><![CDATA[
+        const content = context.getVariable("request.content")
```

Comment:
It's generally easier to write scripts like this in dedicated files in the `resources` dir, then load them here using the `<ResourceURL` tag.

Reply:
Ok! I'll swap this over once I've got it wired up.

Reply:
This has been swapped over to using a resource now.